include/variant_parser/variant_record.hpp

```diff
@@ -66,16 +66,21 @@ class variant_header
      *
      * \tparam stream_type - a stream to print the output to
      *
+     * \param[in, out] references_lengths - a vector of \@SQ header tags tuples: SN and LN
```

Comment:
see my comment above about the description of this parameter

Reply:
Also: Why is that [in, out] and not just [in]?

Reply:
Well seen that was a copy paste mistake.